# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/audioipc/src/messagestream_unix.rs
# Commit: f95a1c5ef0b1
# Full Hash: f95a1c5ef0b1652e911f6ff96c6c38796856e70e
# Author: Matthew Gregan <kinetik@flim.org>
# Date: 2021-08-13 09:27:46
# Regressor Bug: 1724141
# File Overlap Count: 6
# Description:
#   Bug 1724141 - Update audioipc to 8bb1a227.  r=chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D121826
# ==============================================================================

diff -r c9c0a125e6b2 -r f95a1c5ef0b1 third_party/rust/audioipc/src/messagestream_unix.rs
--- a/third_party/rust/audioipc/src/messagestream_unix.rs	Thu Aug 12 21:12:32 2021 +0000
+++ b/third_party/rust/audioipc/src/messagestream_unix.rs	Thu Aug 12 21:28:03 2021 +0000
@@ -26,7 +26,7 @@
     }
 
     #[allow(clippy::missing_safety_doc)]
-    pub unsafe fn from_raw_fd(raw: super::PlatformHandleType) -> MessageStream {
+    pub unsafe fn from_raw_handle(raw: super::PlatformHandleType) -> MessageStream {
         MessageStream::new(net::UnixStream::from_raw_fd(raw))
     }
 